python/lsst/dax/imgserv/api_soda.py

```diff
@@ -0,0 +1,168 @@
+# LSST Data Management System
+# Copyright 2017-2019 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+
+"""
+This module implements the REST interface for Image Cutout Service.
+Corresponding URI: /image
+
+"""
+import os
+import traceback
+import tempfile
+import json
+
+from flask import Blueprint, make_response, request, current_app
+from flask import render_template, send_file, jsonify
+from werkzeug.exceptions import HTTPException
+
+import lsst.log as log
+import lsst.afw.image as afwImage
+
+from .vo.imageSODA import ImageSODA
+
+
+app = Blueprint("api_image_soda", __name__, static_folder="static",
+                           template_folder="templates")
+
+def load_imgserv_config(config_path, metaserv_url):
```

Comment:
I can see that it's a bit confusing for now, but it'll be taken cared of once we remove v1 code entirely.  Functionally, since the scope of the function is at file or module level, there is no issue.